src/coreclr/scripts/superpmi.py

```diff
@@ -1500,6 +1505,10 @@ def save_repro_mc_files(temp_location, coreclr_args, artifacts_base_name, repro_
     """ For commands that use the superpmi "-r" option to create "repro" .mc files, copy these to a
         location where they are saved (and not in a "temp" directory) for easy use by the user.
     """
+
+    if not coreclr_args.produce_repro:
+        pass
```

Comment:
This doesn't do anything. Do you want `pass` to be `return`? Or maybe don't do anything here since it won't find any .mc files.

Reply:
ah yes....`return`.

Reply:
sure, I think I will just remove this because it won't find any mc files. That's why it worked as expected locally when i was testing.